internal/pkg/argocdapp/reinstall.go

```diff
@@ -0,0 +1,34 @@
+package argocdapp
+
+import (
+	"log"
+	"os"
+
+	"github.com/mitchellh/mapstructure"
+)
+
+// Renstall an ArgoCD app .
+func Renstall(options *map[string]interface{}) (bool, error) {
+	var param Param
+	err := mapstructure.Decode(*options, &param)
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	file := "./app.yaml"
+
+	//delete resource
+	kubectlDelete(file)
+	
+	//remove app.yaml file
+	errRemove := os.Remove(file)
+	if errRemove != nil {
+		log.Fatal(errRemove)
```

Comment:
I think err is better than errRemove. and use
```
return false, err
```
instead of Fatal()

Reply:
done